tests/unit/_base/all.js

```diff
@@ -4,6 +4,7 @@ define([
 	'./array',
 	'./connect',
 	'./declare',
+	'./declare-strict',
```

Comment:
They shouldn't run in IE8, but not because it lacks strict mode support. (IE9 also lacks it, but they're fine to run there -- in fact they should.)

The reason is because IE8 and previous has significant JScript bugs with named function expressions, which may invalidate this set of tests.